api/onboarding/urls.py

```diff
@@ -0,0 +1,25 @@
+from common.core.utils import is_oss
+from django.urls import path
+
+from onboarding.views import (
+    ReceiveSupportRequestFromSelfHosted,
+    send_onboarding_request_to_saas_flagsmith_view,
+)
+
+app_name = "onboarding"
+
+urlpatterns = [
+    path(
+        "request/receive/",
+        ReceiveSupportRequestFromSelfHosted.as_view(),
+        name="receive-onboarding-request",
+    ),
+]
+if is_oss():
+    urlpatterns.append(
+        path(
+            "request/send/",
+            send_onboarding_request_to_saas_flagsmith_view,
+            name="send-onboarding-request",
+        ),
+    )
```

Comment:
Right, I guess this negates my comment above. I suppose the logic here is that we want the FE to make an API request to the local API, which makes the request out to Flagsmith SaaS. I guess that makes sense.